tests/nn/misc/test_wrap.py

```diff
@@ -0,0 +1,138 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
when you merge with master, you need to add this file to tests/ci_test_list1.txt to make sure one of the GPU testers will run it in CI. Should be simple one line change. You can run `bash tests/ci_test_list_check.sh` to verify locally.

If you move wrap into nn/wrap, please move this file accordingly as well.